userguide/tutorials/getting_started.adoc

```diff
@@ -284,56 +284,106 @@ The next step is to configure Kill Bill. For this, you need to follow the steps
+
+. You should have a database called *kaui* (Or whatever name you have specified in step 1) with the following tables:
+
+ kaui_users
+ kaui_tenants
+ kaui_allowed_users
+ kaui_allowed_user_tenants
+
+
+==== Kaui Configuration
+
+Finally, Kaui needs to be configured. For this, you need to follow the steps given below:
+
+. Download the current version of the Kaui war from https://search.maven.org/search?q=a:kaui-standalone[Maven Central].
+
+. Ensure that Tomcat server is stopped.
+
+. Place the downloaded war file at *TOMCAT_HOME/webapps*. So, *webapps* folder should look like this:
+
+ $TOMCAT_HOME/webapps
+     --ROOT.war
+     --kaui-standalone-2.0.5.war
+
+. Open *TOMCAT_HOME/conf/catalina.properties file*. Add the following database properties  related to Kaui at the end of this file (Use appropriate values as per your database):
+
+ kaui.url=http://127.0.0.1:8080
+ kaui.db.url=jdbc:mysql://127.0.0.1:3306/kaui
+ kaui.db.username=killbill
+ kaui.db.password=killbill
+
+. Start Tomcat using the script *TOMCAT_HOME/bin/startup.sh* or *TOMCAT_HOME/bin/startup.bat*
+(Depending on your operating system)
+
+. Verify that there are no errors in the Tomcat logs at  *TOMCAT_HOME/logs/catalina.log*
+
+. Verify that there are no errors in the Kill Bill logs on the console and that the logs display a line which states that  *Kill Bill server has started*
+
+. Open a browser and type http://localhost:8080/kaui-standalone-2.0.5. This should display the following sign in page: image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/kaui_sign_in.png[align=center]
+
+. Sign in with `admin/password`. This should display the following page:
+
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/kaui_after_sign_in.png[align=center]
 
 ==== Customizing Log File Path
 
 The steps above configure the application so that the Kill Bill logs are displayed on the console. You can however customise this to save the logs in a separate log file. In order to set this up, you need to follow the steps given below:
 
 . Create a file called `logback.xml` as follows:
 
-  <configuration scan="true" scanPeriod="30 seconds">
-  <jmxConfigurator />
-  <property name="LOGS_DIR" value="<log_file_path>" />
-  <conversionRule conversionWord="maskedMsg" converterClass="org.killbill.billing.server.log.obfuscators.ObfuscatorConverter" />
-  <appender name="MAIN" class="ch.qos.logback.core.rolling.RollingFileAppender">
+ <?xml version="1.0" encoding="UTF-8"?>
+<configuration scan="true" scanPeriod="30 seconds">
+   <jmxConfigurator />
+   <property name="LOGS_DIR" value="F:/killbill/logs" />
+   <conversionRule conversionWord="maskedMsg" converterClass="org.killbill.billing.server.log.obfuscators.ObfuscatorConverter" />
+   <appender name="MAIN" class="ch.qos.logback.core.rolling.RollingFileAppender">
       <file>${LOGS_DIR:-./logs}/killbill.out</file>
```

Comment:
Just to confirm: With this approach, we will end up seeing both KB and KAUI logs redirected to `killbill.log` file, can you confirm?

Reply:
I'm not sure, I will verify this and let you know.

Reply:
Updated the document so that killbill and kaui logs are stored in separate files